Library/Homebrew/test/test_cmd_testbot.rb

```diff
@@ -45,9 +45,7 @@ def run
   end
 
   def teardown
-    unless passed?
-      raise "INFO: Previous test failed with ENV['TRAVIS'] = #{ENV["TRAVIS"].inspect}"
-    end
+    raise "INFO: Previous test failed with ENV['TRAVIS'] = #{ENV["TRAVIS"].inspect}" unless passed?
```

Comment:
Better to try and keep this below 80 chars by e.g. splitting this into an`if .. end`.